.env.production

```
@@ -0,0 +1,8 @@
+VITE_STOMP_TEST_MODE=false
+VITE_BACK_URL=https://i13a505.p.ssafy.io
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_BACK_URL key should go before the VITE_STOMP_TEST_MODE key

(UnorderedKey)